README.md

```diff
@@ -16,9 +16,9 @@
 For NUS students, you can access your IVLE LAPI key [here](https://ivle.nus.edu.sg/LAPI/default.aspx).
 
 ## For Windows Users
-In package.json, change line 19:
-"start-js": "rm -r coverage; BROWSER=none PORT=80 react-scripts-ts start",
-to:
+In package.json, change line 19: 
+"start-js": "rm -r coverage; BROWSER=none PORT=80 react-scripts-ts start", 
+to: 
```

Comment:
Could you remove the trailing spaces here

Reply:
line break not displaying properly so i added backslash